packages/ember-glimmer/lib/helpers/inline-if.js

```diff
@@ -0,0 +1,38 @@
+/**
+@module ember
+@submodule ember-templates
+*/
+
+import { toBool as emberToBool } from './if-unless';
+import { assert } from 'ember-metal/debug';
+
+/**
+  The inline `if` helper conditionally renders a single property or string.
+  This helper acts like a ternary operator. If the first property is truthy,
+  the second argument will be displayed, if not, the third argument will be
+  displayed
+  ```handlebars
+  {{if useLongGreeting "Hello" "Hi"}} Alex
+  ```
+  You can use the `if` helper inside another helper as a subexpression.
+  ```handlebars
+  {{some-component height=(if isBig "100" "10")}}
+  ```
+  @method if
+  @for Ember.Templates.helpers
+  @public
+*/
+export default function inlineIf(args) {
+  assert(
+   'The inline form of the `if` and `unless` helpers expect two or ' +
+   'three arguments, e.g. `{{if trialExpired \'Expired\' expiryDate}}` ',
+   args.length === 2 || args.length === 3
+ );
+
+  if (emberToBool(args[0])) {
+    return args[1];
+  } else {
+    let falsyArgument = args[2];
+    return falsyArgument === undefined ? '' : falsyArgument;
```

Comment:
yes, this was added to allow the moved `it can omit the falsy argument` test to pass. I'll modify this test to be `@htmlbars` and add a TODO as you suggest.